worker/Dockerfile.template

```diff
@@ -17,13 +17,13 @@ COPY bin bin
 
 RUN npm run build
 
-FROM balenalib/%%BALENA_ARCH%%-alpine-node:12.19.1-3.12-run-20201211
-
 ENV UDEV 1
+FROM balenalib/%%BALENA_ARCH%%-alpine:3.12-run-20211030
```

Comment:
@jakogut I shouldn't have approved this, it effectively makes udev unavailable in the runtime container and cannot flash SD cards in testbot.